tests/foreman/ui/test_host.py

```diff
@@ -1886,7 +1901,7 @@ def test_positive_set_multi_line_and_with_spaces_parameter_value(
 
 
 @pytest.mark.pit_client
-@pytest.mark.rhel_ver_match('[^6].*')
+@pytest.mark.rhel_ver_match('[7,8,9]')
```

Comment:
Here we could benefit from the changes made as a part of #19079 and change the regex to `'^[0-9]+$'` to include RHEL 10 too.

Reply:
agree, but let me do it separately, as that would be something to do in master as well